<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiDate</class>
 <widget class="QWidget" name="PiDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PiDate</string>
  </property>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>611</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>116</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="date">
    <date>
     <year>2095</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="day10up">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-up.png</normaloff>go-up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="day1up">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-up.png</normaloff>go-up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="month1up">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-up.png</normaloff>go-up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="year10up">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-up.png</normaloff>go-up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="year1up">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-up.png</normaloff>go-up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="day10down">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-down.png</normaloff>go-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="day1down">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-down.png</normaloff>go-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="month1down">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-down.png</normaloff>go-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="year10down">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-down.png</normaloff>go-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="year1down">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>go-down.png</normaloff>go-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suchen!</string>
   </property>
  </widget>
  <widget class="QLabel" name="output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>751</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="searchall">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Alle Daten suchen...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
